scripts/publish-npm.js

```diff
@@ -117,6 +117,13 @@ artifacts.forEach((name) => {
   }
 });
 
+// don't publish `ReactAndroid` to npm
+const pak = JSON.parse(fs.readFileSync('package.json'));
+
+pak.files = pak.files.filter(filename => filename !== 'ReactAndroid');
+
+fs.writeFileSync('package.json', JSON.stringify(pak, null, 2));
```

Comment:
JSON.stringify(packageJson, null, 2).to(`package.json`);

IF we are using shelljs, better not to mix node api with it unless necessary